### DISCUSSION TITLE
Too Many Redirects on Website with Plus Sign

### DISCUSSION THREAD

**Discussion by peterkimzz** [vercel/community#7347] [Bug Report] [answered]
### Page to Investigate

https://www.eventcat.com/conferences?utm_source=XL8+Audience&utm_campaign=07f0df1f16-EMAIL_CAMPAIGN_2024_05_27_04_32_COPY_05&utm_medium=email&utm_term=0_-1908fd317c-%5BLIST_EMAIL_ID%5D

### Steps to Reproduce

Hi,

I'm trying to send an email by using Mailchimp, but I found that when I open a link that has a plus sign in a utm parameter, it returns `ERR_TOO_MANY_REDIRECTS` error in Chrome.

Here is the URL that doesn't work:
https://www.eventcat.com/conferences?utm_source=XL8+Audience&utm_campaign=07f0df1f16-EMAIL_CAMPAIGN_2024_05_27_04_32_COPY_05&utm_medium=email&utm_term=0_-1908fd317c-%5BLIST_EMAIL_ID%5D

Here is the URL that works: (plug sign -> hypen)
https://www.eventcat.com/conferences?utm_source=XL8_Audience&utm_campaign=07f0df1f16-EMAIL_CAMPAIGN_2024_05_27_04_32_COPY_05&utm_medium=email&utm_term=0_-1908fd317c-%5BLIST_EMAIL_ID%5D

I don't use any redirect options in `vercel.json` file.

**Comment by pawlean** [comment, marked as answer]
Hi @peterkimzz!

Thanks for your post. I'm not familiar with this set-up at all, but it's really interesting. Thanks for sharing, I learned something new 😄 

It sounds like the issue with the URL containing a plus sign (+) in the utm_source parameter resulting in an` ERR_TOO_MANY_REDIRECTS` error may be due to how the server interprets the + sign. In URL encoding, a plus sign can be interpreted as a space, which might cause unexpected behavior in the server's redirection logic.

To avoid this issue, you can replace the plus sign with its URL-encoded equivalent `%2B`. Here's an example:

```
https://www.eventcat.com/conferences?utm_source=XL8%2BAudience&utm_campaign=07f0df1f16-EMAIL_CAMPAIGN_2024_05_27_04_32_COPY_05&utm_medium=email&utm_term=0_-1908fd317c-%5BLIST_EMAIL_ID%5D
```

For me, this renders the page fine.

**Reply by peterkimzz** [reply]
@pawlean Thanks for quick response. So you meant that is it not an intentional behavior of Vercel?

[a comment by an automated account is not shown]
